### DISCUSSION TITLE
Must Reflect Changes On Atri Editor

### DISCUSSION THREAD

**Discussion by Akarsh-1503** [Atri-Labs/atrilabs-engine#585] [Discuss: DX Improvement Ideas]
It never picks up code written directly in VS Code. When  the code is written then instead of changing things only on PORT 4005 it must also reflect on PORT 4002.

After restarting the server it asks for Build & Run which automatically converts the code back to normal, as it was done by Atri Editor at PORT 4002. 

Conclusion:
       Changes must retain even after restarting the server.

**Comment by cruxcode** [comment]
As of now, you might have to rely on visual builder to create your website. You can write css code though. Please watch the tutorial video on how to use the editor and custom css if needed from our YouTube channel.

**Comment by cruxcode** [comment]
In version 1.0.0 we will have this functionality where you can easily write react components. Thanks for pointing it out.
